src/components/dialogs/pick-entity-or-conversion.vue

```diff
@@ -96,56 +119,76 @@ export default class PickEntityOrConversionDialog extends Vue {
     }
   }
 
-  get conversionTypes () {
-    return newReqModule.conversionTypes
-  }
-
   get entity_type_cd () {
-    return newReqModule.entity_type_cd
+    return this.getEntityTypeCd
   }
 
   set entity_type_cd (value) {
-    newReqModule.mutateEntityType(value)
-  }
-
-  get isConversion () {
-    return (newReqModule.request_action_cd === 'CNV')
```

Comment:
You replaced this with `getIsConversion` but I didn't see where it's declared. Does it replace this code?

Reply:
found it